src/sentry/sentry_metrics/indexer/strings.py

```diff
@@ -189,6 +189,8 @@
     "is_application": PREFIX + 269,
     "platform": PREFIX + 270,
     "os.version": PREFIX + 271,
+    # Performance Score
+    "sentry.score_profile_version": PREFIX + 272,
```

Comment:
For this string, it is unlikely, but:

Before merging this, we have to verify that no user has registered this string before. Otherwise, we break the user's query (two different IDs would point to the same string).

```sql
# metrics db
select * from sentry_stringindexer where string = 'sentry.score_profile_version';

#performance metrics db
select * from sentry_perfstringindexer where string = 'sentry.score_profile_version';
```